crates/indexer/src/models/v2_objects.rs

```diff
@@ -120,35 +108,42 @@ impl Object {
     /// This should never really happen since it's very difficult to delete the entire resource group
```

Comment:
This is no longer true. Unnamed token objects can be burnt and deleted very thoroughly. For most use cases using aptos_token, we'll see clean deletions so let's make sure we handle this case well (can be in a separate PR though).

Reply:
Yea I've seen this already so I'll change the comments.